### DISCUSSION TITLE
More width for logs in Actions please 👀

### DISCUSSION THREAD

**Discussion by landsman** [community/community#19029] [Actions]
Why I have to see this sidebar all time when I scrooling?
It doesn't make sense.

Let's enable us to hide it (floating arrow icon).

What developer looking for is the big width for logs, because they are looong.
Try it on some Java/Kotlin project on your own.

<img width="1553" alt="Screenshot 2022-06-21 at 11 28 59" src="https://user-images.githubusercontent.com/3184228/174766859-7c781435-266d-43ed-84ee-938c4da66af9.png">

**Comment by ghost** [comment]
As a temporary band-aid solution you could overwrite some css rules to have a cleaner view of these logs. No extensions are even required when it comes to Safari.

![image](https://user-images.githubusercontent.com/21268739/174789312-53bdc3a7-0edb-454d-818c-b35d3335ba0f.png)

This solution ☝️ will apply CSS overwrites on _all sites_ you visit though, which is a downside if you can't find a uniquely named enough class name that you're sure would only affect the Github log window.

Alternatively, you could use https://cascadea.app/, which allows you to set site-specific css rules.

This CSS overwrite worked for me:

```css
.js-checks-log-display-container {
  overflow: scroll;
}

.js-checks-log-display-container div {
  width: max-content;
}
```
